protocol-dashboard/src/store/index.ts

```diff
@@ -55,7 +55,7 @@ export const getBackupClient = () => {
 
 export const hasBackupClient = !!gqlBackupUri
 
-const aud = new Audius()
+export const aud = new Audius()
```

Comment:
Elsewhere in the code we just do window.aud since it's assigned in the line below. Prob no need to export this.

Reply:
i kinda like the direct import in case window is doing weird things and type-safety, but for consistency that's fine w/ me!